testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestExternalHiveTable.java

```diff
@@ -151,6 +151,17 @@ public void testCreateExternalTableWithInaccessibleSchemaLocation()
         onTrino().executeQuery(format("CREATE TABLE %s.%s.%s WITH (external_location = '%s') AS SELECT * FROM tpch.tiny.nation", HIVE_CATALOG_WITH_EXTERNAL_WRITES, schema, table, tableLocation));
     }
 
+    @Test
+    public void testExternalTablePurgeProperty()
```

Comment:
We avoid product tests as much as possible. Please consider using integration test, e.g. `TestHive3OnDataLake`.